client/js/components/StatsLeaderboardCell.vue

```diff
@@ -28,7 +28,7 @@
         </v-list>
         <h3>Win Rate</h3>
         <v-list :data-win-rate="`${username}-week-${week}`">
-          {{ winRatePercentage }}
+          {{ winRatePercentage }} {{ winRateText }}
```

Comment:
Let's combine these into one higher level computed prop that concatenates them

Reply:
Done